docs/sphinx/examples/python/tutorials/readout_error_mitigation.ipynb

```diff
@@ -29,7 +29,60 @@
    "execution_count": 1,
    "id": "04b58854-7dab-4f1b-8c59-d8b9506bce44",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
This output should probably be cleared. Leaving it in there just muddies the docs page in my opinion.

And I'm not really sure why changes to the "readout_error_mitigation.ipynb" tutorial are being done under the "Adding AFQMC tutorial". Are they related to each other?